serializations.md

```diff
@@ -770,6 +771,23 @@ version 2, from Fortuna release
 ]
 ```
 
+#### Channel client reconnect transaction
+
+The channel client reconnect transaction is used only when a Websocket client
+wants to reconnect to an already running FSM. It cannot be introduced into the
+mempool. The elements of the transaction are:
+* The channel id of the channel that the client wants to connect to
+* The role (`initiator` or `responder`) of the FSM instance in question
+* The public key of the client; must correspond to the private key used for
```

Comment:
```suggestion
* The public key of the client; must correspond to the authentication used for
```

Reply:
or maybe "authentication method" (but this breaks the 80 chars so a bit harded for a suggestion)